test/unit/spec/PrimaryAuth_spec.js

```diff
@@ -1920,36 +1920,6 @@ function (Q, OktaAuth, LoginUtil, Okta, Util, AuthContainer, PrimaryAuthForm, Be
             expectErrorEvent(test, 429, 'API call exceeded rate limit due to too many requests.');
           });
       });
-      itp('shows an error if authClient returns with an error that is plain text', function () {
```

Comment:
Why are we removing this test? authClient could still return an error that is plain text right?

Reply:
They authClient should always return the parsed object, while in this tests we were mocking that returns a string, which we don't parse anymore in the new way and which should throw this error: https://github.com/okta/okta-signin-widget/pull/692/files#diff-a2204dc4da481825a3d452cdd6c378ccR65.

But good point, I should add test for that 👍